pkg/daemon/ceph/agent/controller.go

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Agreed but I still don't see anything on the horizon to remove it. In case this does not make it in 1.8 I decided to write the controller, it's simple anyway. If we are sure to remove it I can open a separate PR right away and remove the controller from the PR. What do you think?

Reply:
Sounds good since it was simple and already created, when we're ready to remove it will just be a separate simple PR.